### ISSUE TITLE
Accessibility: PropertyGrid: An onscreen element must not have a null BoundingRectangle property

### ISSUE THREAD

**Issue by Olina-Zhang** [dotnet/winforms#5656]
* .NET Core Version: 
.Net SDK 6.0.100-preview.7.21379.14

* Have you experienced this same bug with .NET Framework?: 
Yes

**Minimal repro:**
1. Open attached application in VS,  with PropertyGrid control 
[WinFormsApp15.zip](https://github.com/dotnet/winforms/files/7087962/WinFormsApp15.zip)
2. Build and run it
3. In AccessibilityInsight or Inspect to detect propertyGrid 

**Actual behavior:**
1. AccessibilityInsight detects an error about "An onscreen element must not have a null BoundingRectangle property" for Text property
![image](https://user-images.githubusercontent.com/26474449/131605754-54f1aaa8-2372-4dd1-b4dd-04a7525d6230.png)
2. In Inspect, for Text property, there is missing BoudingRectangle property, and other properties are incorrect for Role, State, Value, Value.Value.
![image](https://user-images.githubusercontent.com/26474449/131606538-d965ff75-5daf-4346-8f4b-5e4ed7ad8e95.png)

**Expected behavior:** Following results are gotten when clicking text edit area, then use accessibility tools to detect
1. AccessibilityInsight detects no error:
![image](https://user-images.githubusercontent.com/26474449/131606589-e4210001-b92b-4da0-937c-fd3799e5adf5.png)
2. In inspect, it has BoudingRectangle property, and other properties: Role, State, Value, Value.Value are correct:
![image](https://user-images.githubusercontent.com/26474449/131606689-0b183ccc-e751-41df-82e3-a9d99b839f68.png)

**Comment by Jenna-Zheng** [comment]
Verified this issue with .NET SDK 6.0.100-rc.2.21465.13, it is fixed.
1. AccessibilityInsight detects no error:
![image](https://user-images.githubusercontent.com/64949616/133558820-97214c04-cfd0-4355-9471-e02dbb814d83.png)
2. In inspect, it has BoudingRectangle property, and other properties: Role, State, Value, Value.Value are correct:
![image](https://user-images.githubusercontent.com/64949616/133559446-43ff0cb3-c676-433c-b65f-1b2df5071222.png)

**Comment by Lydia-Shi** [comment]
Verified this issue with .NET SDK 7.0.100-alpha.1.21479.26, this issue is fixed.
